backend/src/main/java/com/votogether/domain/post/service/PostService.java

```diff
@@ -365,5 +369,50 @@ private <T, R> List<R> transformElements(final List<T> elements, final Function<
                 .toList();
     }
 
+    @Transactional(readOnly = true)
+    public List<PostRankingResponse> getRanking() {
+        final List<Post> posts = postRepository.findAllByClosingTypeAndSortTypeAndCategoryId(
+                PostClosingType.ALL,
+                PostSortType.HOT,
+                null,
+                PageRequest.of(0, BASIC_PAGING_SIZE)
+        );
+
+        final Map<Post, Integer> rankings = calculateRanking(posts);
+
+        return rankings.entrySet().stream()
+                .sorted(Comparator.comparingInt(entry -> entry.getValue()))
+                .map(entry ->
+                        new PostRankingResponse(
+                                PostCompactResponse.of(entry.getKey()),
+                                entry.getValue()
+                        )
+                )
+                .toList();
+    }
+
+    private Map<Post, Integer> calculateRanking(final List<Post> posts) {
+        final List<Post> sortedPosts = posts.stream()
+                .sorted(Comparator.comparingLong(post -> -post.getTotalVoteCount()))
+                .toList();
```

Comment:
`P2`
```suggestion
    private Map<Post, Integer> calculateRanking(final List<Post> posts) {
        final List<Post> sortedPosts = posts.stream()
                .sorted(Comparator.comparingLong(Post::getTotalVoteCount).reversed())
                .toList();
```
내림차순으로 정렬하는 경우 이렇게 변경하는 것은 어떨까요?